tests/api_test.py

```diff
@@ -608,10 +618,12 @@ def test_create_unknown_participant(flask_client):
     code = data.pop('code')
     id_ = data.pop('id')
     participant = Participant.query.filter_by(name=name).first()
+    newdle = Newdle.query.filter_by(code='dummy').first()
     assert data == {'answers': {}, 'auth_uid': None, 'email': None, 'name': name}
     assert Participant.query.count() == num_participants + 1
     assert participant.code == code
     assert participant.id == id_
+    assert newdle.last_update > before_update
```

Comment:
I wonder if it wouldn't be better to add some separate test cases which only test that the last update timestamp gets updated whenever relevant actions are performed.

Right now this is tested by test cases that aren't really related to this..

Reply:
Sure, I'll remove them from here and add separate tests.